azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/ListMojo.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.function;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.maven.plugins.annotations.Mojo;
+
+import java.io.InputStream;
+
+/**
+ * The list mojo is used to provide Java Azure Functions templates information,
+ * which is used by other tools such as VS Code Azure Functions extension.
+ */
+@Mojo(name = "list")
+public class ListMojo extends AbstractFunctionMojo {
+
+    public static final String PRINTING_START = "Printing Java Azure Functions templates information...";
+    public static final String PRINT_END = "Printing Java Azure Functions templates information done.";
+
+    @Override
+    protected void doExecute() throws Exception {
+        info(PRINTING_START);
```

Comment:
When you print logs around the json file content, the caller will not be able to consume the output directly. Please remove the begin and end log statement and just print the content.

Reply:
Updated:

using 

`>> templates begin <<` & `>> templates begin <<` to notify the start and end of the output.